cores/arduino/USB/USBAPI.h

```diff
@@ -128,6 +127,8 @@ class Serial_ : public Stream
 	using Print::write; // pull in write(str) from Print
 	operator bool();
 
+	size_t readBytes(char *buffer, size_t length);
```

Comment:
Since it is not `virtual` it won't necessarily be called if `SerialUSB` is passed in to code that accepts a `Stream` object. Just something to keep in mind.